### ISSUE TITLE
DAC 33 - Medium Priority WCAG Level AA - Text Spacing

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#33]
<img width="275" alt="DAC_Text_Spacing_01" src="https://user-images.githubusercontent.com/75850106/155572253-677876f6-05ed-4265-a0f3-9c126328c948.png">

### Issue
Low vision users that modify the text spacing for an easier reading experience will find that text in the graphs begins to overlap. This makes it difficult for the user to read the content. This happens in multiple instances on the page.

**Low vision comments:**
“Looking at the chart in the top right corner of the map, when looking at the page with text spacing enabled, the text at the top of the chart cuts off data, the ‘30%’ is missing off the top of this chart.”

### Suggested Solution
Ensure that overriding text spacing does not make text overlap content.

- Line height (line spacing) to at least 1.5 times the font size; 
- Spacing following paragraphs to at least 2 times the font size; 
- Letter spacing (tracking) to at least 0.12 times the font size; 
- Word spacing to at least 0.16 times the font size.
